### PR TITLE
Tweaks to 2 computer example

### PR DESCRIPTION
**Pull request by 4c656554**
1. I had async error so encapsulated in aync funcs
2. Added await to node.connect func
3. Added example write for a noiseSocket (took a while to find this)
4. Added console log for receipt of data at first computer
5. Log keys in hex format

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,35 +31,51 @@ Then on one computer listen for connections
...
-// publicKey here is keyPair.publicKey from above
-const noiseSocket = anotherNode.connect(publicKey)
+start()
 
-noiseSocket.on('open', function () {
-  // noiseSocket fully open with the other peer
-})
+async function start () {
+    // publicKey here is keyPair.publicKey from above
+    const noiseSocket = await anotherNode.connect(publicKey)
```
Comment (mafintosh) [inline README.md:67]: no await here

Comment (4c656554) [reply README.md:67]: ah ok,

File: README.md [hunk 2]
```diff
@@ -31,35 +31,51 @@ Then on one computer listen for connections
...
+    console.log('Local public key', noiseSocket.publicKey.toString('hex')) // same as keyPair.publicKey
 
-// make a ed25519 keypair to listen on
-const keyPair = DHT.keyPair()
+    //process.stdin.pipe(noiseSocket).pipe(process.stdout) // pipe somewhere
 
-// this makes the server accept connections on this keypair
-await server.listen(keyPair)
+    // log received data:
+    noiseSocket.on('data',(data) => {
```
Comment (mafintosh) [inline README.md:47]: missing space after `'data',`

File: README.md [hunk 3]
```diff
@@ -31,35 +31,51 @@ Then on one computer listen for connections
...
-process.stdin.pipe(noiseSocket).pipe(process.stdout)
+    noiseSocket.on('open', function () {
+      // noiseSocket fully open with the other peer
+      // send a test message:
+      noiseSocket.write(Buffer.from('test'))
+    })
+
+    // pipe it somewhere like any duplex stream
+    // process.stdin.pipe(noiseSocket).pipe(process.stdout) 
+
```
Comment (mafintosh) [inline README.md:77]: remove extra newline here

File: README.md [hunk 4]
```diff
@@ -31,35 +31,51 @@ Then on one computer listen for connections
 // create a server to listen for secure connections
 const server = node.createServer()
 
-server.on('connection', function (noiseSocket) {
-  // noiseSocket is E2E between you and the other peer
-  // pipe it somewhere like any duplex stream
+start()
```
Comment (mafintosh) [inline README.md:34]: understand why, but let's not wrap in an async start function here and instead perhaps make a comment for that :)

**Comment by mafintosh** [comment]
Thanks! Added some comments

File: README.md (hunk 4 above)
Comment (4c656554) [reply README.md:34]: ok, will edit

File: README.md [hunk 5]
```diff
@@ -31,35 +31,51 @@ Then on one computer listen for connections
...
+async function start () {
+    // publicKey here is keyPair.publicKey from above
+    const noiseSocket = await anotherNode.connect(publicKey)
 
-// pipe it somewhere like any duplex stream
-process.stdin.pipe(noiseSocket).pipe(process.stdout)
+    noiseSocket.on('open', function () {
+      // noiseSocket fully open with the other peer
+      // send a test message:
+      noiseSocket.write(Buffer.from('test'))
```
Comment (mafintosh) [inline README.md:72]: you can write immediately, no need to wait for open

**Pull request closed without merging**